rpm.pc.in

```diff
@@ -8,7 +8,7 @@ Name: RPM
 Description: RPM Package Manager
 Version: @VERSION@
 URL: http://rpm.org
-# Requires: 
+Requires.private: @ZSTD_REQUIRES@
```

Comment:
If this is linked in from librpmio, there's no reason to add `Requires.private`, right? We never did this for any of the others...